processing/src/test/java/org/apache/druid/segment/TestNullableDoubleColumnSelector.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.segment;
+
+import org.apache.druid.common.config.NullHandling;
+
+public class TestNullableDoubleColumnSelector extends TestDoubleColumnSelector
+{
+  private final Double[] doubles;
+
+  static {
+    NullHandling.initializeForTests();
+  }
+
+  private int index = 0;
+
+  public TestNullableDoubleColumnSelector(Double[] doubles)
+  {
+    this.doubles = doubles;
+  }
+
+  @Override
+  public double getDouble()
+  {
+    return doubles[index] == null ? NullHandling.ZERO_DOUBLE : doubles[index];
+  }
+
+  @Override
+  public boolean isNull()
+  {
+    return doubles[index] == null;
```

Comment:
Probably this should be `return NullHandling.replaceWithDefault() ? false : doubles[index] == null`.

Reply:
Same for other column selectors.

Reply:
Done.